genai-perf/tests/test_csv_exporter.py

```diff
@@ -335,3 +341,151 @@ def test_invalid_goodput_csv_output(
                 f"Expected file {expected_filename} not found in written data."
             )
         assert returned_data[-1] == expected_content
+
+    def test_triton_telemetry_metrics_output_success(
+        self, monkeypatch, mock_read_write: pytest.MonkeyPatch
+    ) -> None:
+        argv = [
+            "genai-perf",
+            "profile",
+            "-m",
+            "model_name",
+            "--service-kind",
+            "triton",
+            "--streaming",
+            "--server-metrics-url",
+            "http://tritonserver:8002/metrics",
+        ]
+        monkeypatch.setattr("sys.argv", argv)
+        args, _ = parser.parse_args()
+
+        metrics = LLMMetrics(
```

Comment:
I see a lot of tests with these duplicate lines... is there a way to avoid this? (maybe using `pytext.fixture` could help?)